src/__tests__/CompareResults/beta/ResultsView.test.tsx

```diff
@@ -178,10 +175,12 @@ describe('Results View', () => {
       />,
     );
 
-    const expandButtons = screen.getAllByTestId('expand-revision-button');
+    const expandButtons = await screen.findAllByTestId(
+      'expand-revision-button',
+    );
     await user.click(expandButtons[0]);
-    const expandedContent = await waitFor(() =>
```

Comment:
I also used this opportunity to remove `waitFor` calls that can be replaced with a `find`.